src/java.base/share/classes/sun/security/ssl/TrustManagerFactoryImpl.java

```diff
@@ -105,30 +105,6 @@ protected TrustManager[] engineGetTrustManagers() {
         return new TrustManager[] { trustManager };
     }
 
-    /*
```

Comment:
I think you can also remove the `import java.io.*` on line 28 now that this block of code has been removed.